docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -128,6 +129,16 @@ Change alerts, or alerts that use a percent change threshold, are useful when yo
 
 ![When the percent change option is selected.](./img/percent-change.png)
 
+### Dynamic Alerts
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+Dynamic alerts can be used when you don't know what threshold to set, but you know you want to be alerted when something is far outside the bounds of normalcy. Sentry will look at the historical data for the given metric and determine if the current data is anomalous. You can select how responsive the alert will be, and whether you want to be alerted when the metric is above, below, or both above and below the expected bounds.
+
+Dynamic alerts do not use the Critical, Warning, and Resolved thresholds like other alert threshold types as you are not setting these yourself. When no more anomalies are detected, the alert will resolve.
```

Comment:
I'm not sure why it matters that "you're not setting these yourself". Can you rephrase what you're trying to get across here or maybe we can just remove it so it's easier to read?

```suggestion
Dynamic alerts do not use the Critical, Warning, and Resolved thresholds. When no more anomalies are detected, the alert will resolve on its own.
```